### DISCUSSION TITLE
Infinite Truchet: Exercise 1

### DISCUSSION THREAD

**Discussion by sharlagelfand** [posit-conf-2023/creative-coding#10] [Show and tell]
- Open the `exercises/05-infinite-truchet/truchet_tiles_examples` directory. These are different examples of truchet tiles. Choose one example.
- Draw four squares on your graphing paper in a 2x2 format (see above).
- Draw four different rotations of your selected truchet tile example. 
- Take a picture of your piece when finished and share it on this Github Discussion

**Comment by Meghansaha** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/67963743/75df337b-24cc-4fef-bf7e-9200b707099f)

**Comment by thomaspara** [comment]
![Untitled](https://github.com/posit-conf-2023/creative-coding/assets/10082327/96c9a627-0d55-4899-a395-ed73d8fa9953)

**Comment by sshaughnessy** [comment]
![truchet1](https://github.com/posit-conf-2023/creative-coding/assets/36434962/b08abb1e-aae8-4368-9ebb-b905ae0040eb)

**Comment by MHanken** [comment]
![PXL_20230918_212104804](https://github.com/posit-conf-2023/creative-coding/assets/49319823/23bc9601-b94b-440d-aa7f-7afe5acb1a48)

**Comment by lwdozal** [comment]
![IMG_0646](https://github.com/posit-conf-2023/creative-coding/assets/19821171/4e66b875-a300-46a8-a709-41c9881cb22b)

**Comment by Kim-s-h** [comment]
![sss](https://github.com/posit-conf-2023/creative-coding/assets/22897443/8ee360b7-48cf-48c6-a5a8-396ac55be261)

**Comment by maxrogow** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/145362959/7db31615-5031-480b-8adb-e80883044536)

**Comment by natalieoshea** [comment]
![image](https://github.com/posit-conf-2023/creative-coding/assets/10425075/d7e211a2-173d-4b6c-8a53-24fba1823278)

**Comment by dougtommet** [comment]
![IMG_1640](https://github.com/posit-conf-2023/creative-coding/assets/7944225/987eac90-a326-42c9-a23a-fb6a2898614d)
